docs/dev/perl5/prep/inputs/5.16.2.source.txt

```diff
@@ -0,0 +1,40 @@
+The Perl 5 development team is gratified to announce the release of
```

Comment:
These "working files" in the repo (and the .html) under prep are my biggest issues with it.  I can look the other way on the formatting comments above.